src/components/data/attributetable/attributetable.scss

```diff
@@ -1,10 +1,61 @@
 .mykn-attributetable {
   display: table;
+  width: 100%;
+
+  &--compact {
+    width: unset;
+
+    & .mykn-attributetable__body {
+      grid-template-columns: 1fr;
+      row-gap: 0;
+    }
+
+    & .mykn-attributetable__divider {
+      display: none;
+    }
+
+    & .mykn-attributetable__row {
+      border: none;
+      padding: 0;
+
+      &:last-of-type {
+        border: none;
+        padding-bottom: 0;
```

Comment:
Please use padding-block-end instead of padding-bottom (don't use anything with left, right, top, bottom if modern alternatives exist)